src/natcap/invest/utils.py

```diff
@@ -603,10 +602,7 @@ def build_lookup_from_csv(
         to_lower (bool): if True, converts all unicode in the CSV,
             including headers and values to lowercase, otherwise uses raw
             string values. default=True.
-        expand_path_cols (list[string])): if provided, a list of the names of
```

Comment:
This doesn't need to be documented here, it's passed directly to `read_csv_to_dataframe`, so it's just part of `**kwargs`.